build/heartbeat-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o xtrace
+
+until [ ! -f /var/lib/mysql/bootstrap.lock ] && [ ! -f /var/lib/mysql/clone.lock ]; do
+	echo "waiting for MySQL initialization"
+	sleep 10
+done
+
+sleep 60
+
+echo "Started pt-heartbeat"
+pt-heartbeat \
+	--update \
+	--replace \
+	--fail-successive-errors 20 \
+	--check-read-only \
+	--create-table \
+	--database sys_operator \
+	--table heartbeat \
+	--user heartbeat \
+	--password "${HEARTBEAT_PASSWORD}" \
```

Comment:
we will print password in the log

Reply:
Why  do we use env var? If I want to change this  password mysql pod will be restarted :(